frontend/src/pages/org/collection-detail.ts

```diff
@@ -60,8 +65,14 @@ export class CollectionDetail extends LiteElement {
         >
           ${this.collection?.name || html`<sl-skeleton></sl-skeleton>`}
         </h2>
+        ${when(this.isLoading, () => html`
+          <div class="flex justify-center mr-2 p-2">
```

Comment:
This should align things vertically:
```suggestion
          <div class="flex items-center justify-center mr-2 p-2">
```